### PR TITLE
[DOCS] chore: adding landing page content

### PR DESCRIPTION
**Pull request by sstraatemans**

### CODE REVIEW

File: packages/apps/docs/src/components/Breadcrumbs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export const Breadcrumbs: FC<IProps> = ({ menuItems }) => {
         {items.map((item, idx) =>
           idx < items.length - 1 ? (
             <StyledBreadcrumbs.Item key={item.root} asChild>
-              <Link href={`#${item.root}`}>{item.title}</Link>
```
Comment (sstraatemans) [inline packages/apps/docs/src/components/Breadcrumbs/Breadcrumbs.tsx:46]: fix the breadcrumbs

File: packages/apps/docs/src/pages/docs/contribute/index.tsx [hunk 2]
```diff
@@ -1,38 +1,101 @@
...
+              Autonomous Organization, on its public blockchain called dao.init.
+              As Kadena’s ecosystem continues to grow, the creation of a DAO
+              will allow for the broader community to provide their input in a
+              decentralized manner.
+            </Text>
+          </Box>
+          <Button as="a" asChild>
+            <Link href={'/docs/contribute/kadena-dao'}>Start contributing</Link>
+          </Button>
+        </section>
```
Comment (realdreamer) [inline packages/apps/docs/src/pages/docs/contribute/index.tsx:35]: imo this could be a separate docs component

Comment (sstraatemans) [reply packages/apps/docs/src/pages/docs/contribute/index.tsx:35]: good point..maybe this was just lazyness from my side :)

**Review by realdreamer — COMMENTED** [review]
image

Both menu looks same but referring different api specs https://alpha-docs-git-chore-landingpagescontent-kadena-js.vercel.app/docs/build/cryptography-utils/crypto-api

**Review by realdreamer — APPROVED** [review]

**Pull request merged**